backend/de.metas.contracts/src/main/java/de/metas/contracts/impl/CustomerRetentionRepository.java

```diff
@@ -64,13 +68,18 @@
 	private final IInvoiceDAO invoiceDAO = Services.get(IInvoiceDAO.class);
 	private final IContractsDAO contractsDAO = Services.get(IContractsDAO.class);
 	private final ContractInvoiceService contractInvoiceService;
-	
+
+	public I_C_Customer_Retention getById(final CustomerRetentionId customerRetentionId)
```

Comment:
pls add `@NonNull` at least to public methods like this one

Reply:
Done.